example/lib/screen/footer_page.dart

```diff
@@ -1,5 +1,8 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_web_base/flutter_web_base.dart';
+import 'package:flutter_web_base/footer/footer_3/footer_3.dart';
+import 'package:flutter_web_base/footer/footer_4/footer_4.dart';
+import 'package:flutter_web_base/footer/footer_5/footer_5.dart';
```

Comment:
this should not be imported directly it should be added in the `footer.dart` file in the `footer` folder